src/extensions/renderer-api/k8s-api.ts

```diff
@@ -20,6 +21,9 @@ export function isAllowedResource(resource: KubeResource | KubeResource[]) {
   });
 }
 
+export {
+  BaseKubeJsonApiObjectMetadata,
+};
```

Comment:
Nit: Change is unrelated to commit?

Reply:
I cannot easily find the commit message it seems since the web-ui is rather lagy. (I don't plan on responding to all commit message reviews). 

As for this, I export it to keep the extension API full with all mentioned items. See https://github.com/lensapp/lens/pull/4592 for the motivation behind this. I know that the PR is closed but that is only because no one was reviewing it and I felt that it would be more work to get it back up to date then to just recreate it.